src/smc-webapp/editor.coffee

```diff
@@ -1075,6 +1075,14 @@ class exports.Editor
             if @open_file_tabs().length < 1
                 @resize_open_file_tabs()
 
```

Comment:
@williamstein I did not write that code so I have no clue... https://github.com/timothyclemansinsea/smc/blame/454177e2b0d8e55d218b44b6281950d4995dd2e0/src/smc-webapp/editor.coffee#L1075